.gitignore

```diff
@@ -8,3 +8,5 @@
 GNUmakefile
 /.dub
 /tests_extractor
+/dtools_*
+/dub.selections.json
```

Comment:
This should not be ignored for executables.

Reply:
The motivation was that the tools repo is low traffic anyhow and by saving the `dub.selections.json`, we commit ourselves to update it _every time_ there's an important bug fix in libdparse.
Too much effort for little gain in my opinion.

Reply:
Or there’s a new bug introduced in a new version of libdparse that breaks everything. But whatever.